index.tsx

```diff
@@ -124,7 +124,11 @@ export default function PlausibleProvider(props: {
     HTMLScriptElement
   >
 }) {
-  const { enabled = process.env.NODE_ENV === 'production' } = props
+  const {
+    enabled = process.env.NODE_ENV === 'production' &&
+      (!process.env.NEXT_PUBLIC_VERCEL_ENV ||
```

Comment:
I am assuming this `!process.env.NEXT_PUBLIC_VERCEL_ENV` is just in case someone is deploying it outside of Vercel.

Reply:
Exactly.